js/private/translate_pnpm_lock.bzl

```diff
@@ -303,6 +314,12 @@ def _impl(rctx):
         patch_args = rctx.attr.patch_args.get(name, [])[:]
         patch_args.extend(rctx.attr.patch_args.get(versioned_name, []))
 
+        postinstall = rctx.attr.postinstall.get(name)
+        if not postinstall:
+            postinstall = rctx.attr.postinstall.get(versioned_name)
+        elif rctx.attr.postinstall.get(versioned_name):
+            postinstall = "%s && %s" % (postinstall, rctx.attr.postinstall.get(versioned_name))
```

Comment:
hmm. maybe this will be conceptually simpler and more robust if we use different script names for each level
`_rules_js_postinstall_name`, `_rules_js_postinstall_versioned_name`, `_rules_js_postinstall_global` and we run then in the order `global`, `name`, `versioned_name` in the rule

Reply:
Hm I'm mixed on this one. That would mean that each `npm_import` needs to take three postinstall attributes which seems less clean, especially if someone is calling it manually. I don't feel strongly though. Wdyt?

Reply:
Oh. You can differentiate the scope without having different npm_import attributes. But this seems better to discuss as a follow-up as it might be that `&&` is actually very robust and simpler.

Reply:
Oh, I see what you mean in `npm_import`. That a good point. The alternative would be to make it a string_list but that adds complication. Guess we just need some test coverage for `&&` on different platforms.